helix-core/src/main/java/org/apache/helix/controller/stages/MessageGenerationPhase.java

```diff
@@ -53,6 +52,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+
```

Comment:
@desaikomal It occurred to me that you might not be using helix-style-intellij.xml to format the code. It's a requirement for Helix PRs, and you can find guidelines here: https://github.com/apache/helix/wiki/Helix-Code-Style-in-Intellij-IDE :)

Reply:
I am using intellij-code-style but since my tests were failing on local machine while worked on linux-box, i did from that box.